dash/development/_py_components_generation.py

```diff
@@ -274,30 +268,24 @@ def create_docstring(component_name, props, events, description):
                 description=prop['description'],
                 indent_num=0,
                 is_flow_type='flowType' in prop and 'type' not in prop)
-            for p, prop in list(filter_props(props).items())),
-        events=', '.join(events))
+            for p, prop in list(filter_props(props).items())))
 
 
-def parse_events(props):
+def prohibit_events(props):
     """
-    Pull out the dashEvents from the Component props
+    Events have been removed. Raise an error if we see dashEvents or fireEvents
 
     Parameters
     ----------
     props: dict
         Dictionary with {propName: propMetadata} structure
 
-    Returns
+    Raises
     -------
-    list
-        List of Dash event strings
+    ?
     """
-    if 'dashEvents' in props and props['dashEvents']['type']['name'] == 'enum':
-        events = [v['value'] for v in props['dashEvents']['type']['value']]
-    else:
-        events = []
-
-    return events
+    if 'dashEvents' in props or 'fireEvents' in props:
+        raise AttributeError('Events are no longer supported by dash')
```

Comment:
Can we add a custom exception or raise a DeprecationWarning instead.

Reply:
Well, it wasn't deprecated, it was outright removed 😅 but I'm happy to make a custom exception - `ObsoleteEventsError`?

Reply:
I think `exceptions.NonExistantPropException` with a event removal message would fit the bill.

Reply:
I used `NonExistentEventException`, which is the same I use for corresponding [obsolete callback usage](https://github.com/plotly/dash/pull/550/files#diff-228c9aefac729977642672aa1416bacaR724).